openlibrary/templates/search/search_bar.html

```diff
@@ -0,0 +1,4 @@
+<div class="search-div">
+    <input type="text" class="search-input" name="q" size="100" placeholder="$_('Search')" aria-label="Search" autocomplete="off"><!--
+    --><input type="submit" value="" class="search-btn-icon" aria-label="Search submit">
```

Comment:
We don't need to avoid whitespace here, since you have the container as an inline-flex. That's not affected by whitespace.

```suggestion
    <input type="text" class="search-input" name="q" size="100" placeholder="$_('Search')" aria-label="Search" autocomplete="off">
    <input type="submit" value="" class="search-btn-icon" aria-label="Search submit">
```